### DISCUSSION TITLE
Theme in the Readme

### DISCUSSION THREAD

**Discussion by schell** [neovide/neovide#428] [Q&A] [answered]
This project looks great! What theme and font are you using for the screenshots in the readme?

**Comment by Kethku** [comment, marked as answer]
https://github.com/morhetz/gruvbox
And I use Delugia Nerd Font for the font

**Reply by schell** [reply]
Thank you! Nerd Fonts looks great!

**Reply by Marin-Kitagawa** [reply]
Hi @Kethku how to enable the top navigation bar and rounded sections in the bottom-most bar as show in the animated cursors animation?
I installed the `gruvbox` animation and for the configuration, I took the `.gif` file from the [Animation Trail Length](https://github.com/Kethku/neovide/wiki/Configuration#animation-trail-length) converted it to `.mp4` and then paused it and then typed the configuration from this for `airline`. In this `.gif` file I guess it was Windows OS guessing from the title bar with similar-to-Windows Min-Max-Exit buttons. I couldn't get it working. 

By top `Navbar`, I mean the top bar which says `settings.vim` in the top in the same `.gif` file. Could you please say how to enable this bar?

**Reply by Kethku** [reply]
I gave some details here: https://github.com/Kethku/neovide/discussions/633
